code/lib/cli/src/doctor/getDuplicatedDepsWarnings.ts

```diff
@@ -31,9 +31,9 @@ export const allowList = [
 export const disallowList = [
   Object.keys(rendererPackages),
   Object.keys(frameworkPackages),
-  '@storybook/core-events',
+  'storybook/core-events',
   '@storybook/instrumenter',
-  '@storybook/core-common',
+  '@storybook/core/common',
```

Comment:
Just to understand things. The Storybook CLI `storybook` relies on `@storybook/core` currently and nobody else. And since `storybook` only exists once, `@storybook/core` should also only exist once. Is this just a precaution to add it to the list?